drivers/display/display_sdl_bottom.c

```diff
@@ -98,6 +99,56 @@ int sdl_display_init_bottom(uint16_t height, uint16_t width, uint16_t zoom_pct,
 	}
 	SDL_UnlockTexture(*background_texture);
 
+	/* Create ellipse mask texture if rounded mask is enabled */
+	if (rounded_mask) {
+		*mask_texture = SDL_CreateTexture(*renderer, SDL_PIXELFORMAT_ARGB8888,
```

Comment:
Given that SonarCloud is complaining about the complexity of this func. and this is quite an isolated calculation, maybe this new block should be refactored into its own function?